tests/integration/flows/test_multiple_flows.py

```diff
@@ -23,3 +23,22 @@ def test_multiple_flows_not_working(caplog):
     flow = MultipleFlows(name="test", flows_list=list)
     flow.run()
     assert "One of the flows has failed!" in caplog.text
+
+
+def test_multiple_flows_working_one_flow(caplog):
+    list = [["Flow of flows 1 test", "dev"]]
+    flow = MultipleFlows(name="test", flows_list=list)
+    with caplog.at_level(logging.INFO):
+        flow.run()
+    assert "All of the tasks succeeded." in caplog.text
+
+
+def test_multiple_flows_last_not_working(caplog):
+    list = [
```

Comment:
Just a suggestion - it's better not to use the variable name that is in this dictionary `__builtins__.__dict__.keys()`. In this case `flow_list` instead of `list`.